### ISSUE TITLE
Settings for Capturing Radiance vanished

### ISSUE THREAD

**Issue by Alextocode** [Mantan21/Genshin-Impact-Wish-Simulator#109]
Hi there,

in the first release of Wish Simulator for Genshin 5.0 there used to be settings for the Capturing Radiance Mechanism - one could for instance set the percentage for successful capture.

It was suddenly gone few days later and still not back. Can we bring it back? The feature is useful for simulations:

![grafik](https://github.com/user-attachments/assets/6a7bf564-7417-43d1-a788-b65adcb5ef42)

**Comment by AguzzTN54** [comment]
For now, I have decided to disable Capturing Radiance Customization, because Capturing Radiance is not that simple. The mechanism understood by the community when it was first announced was incorrect.

You can refer to [this thread](https://www.reddit.com/r/Genshin_Impact/comments/1f5mfqf/the_new_cn_capturing_radiance_theory/) to see one of the most reasonable explanations of how Capturing Radiance works, based on an analysis of several samples from players' pull histories.

In short, Capturing Radiance is a system that prevents players from losing consecutively too often. Capturing Radiance is guaranteed after a player loses four times in a row.

Even that may not be entirely accurate, as the available samples are still quite limited, which is why I'm waiting for more relevant information before re-enabling the customization for Capturing Radiance

**Comment by Alextocode** [comment]
Understood.
I'm not sure about that though.
I had 3 lost 50:50 in a row and did not see it, my buddy had a capturing radiance after just 1 lost 50:50....
